src/app/util/ContentApp.cpp

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
This is probably the wrong place for this file.  This directory is basically only for code we imported from the original ZCL implementation.  New code should go into more usefully named places, and in particular places where we can enable `-Wconversion`.

Reply:
addressed in https://github.com/project-chip/connectedhomeip/pull/12969